### DISCUSSION TITLE
Class roll call

### DISCUSSION THREAD

**Discussion by arbennett** [HAS-Tools-Fall2022/Course-Materials22#1] [General]
Once you've created your account and I've added you here please make a comment with your name as well as a link to a cute animal picture.

**Comment by arbennett** [comment]
Hi everyone, it's Andrew. It might not be cute, but this is an accurate description of my cat: 
https://i.kym-cdn.com/photos/images/facebook/001/914/143/88a

![](https://i.kym-cdn.com/photos/images/facebook/001/914/143/88a)

**Comment by kaylamc9** [comment]
![panda](https://user-images.githubusercontent.com/111896262/186204056-d07614c0-1d12-4130-b657-d184b9c25297.jpg)
Hi everyone this is Kayla! I really like pandas because they are clumsy. Here is a panda falling on its face

**Comment by nicolecaryl12** [comment]
![th](https://user-images.githubusercontent.com/111896270/186204431-06f9f5e2-943e-4d8b-b8c6-cde03d590f45.jpeg)
Hello!  I am Nicole and otters are my favorite animal. :D

**Comment by lauren-cutler** [comment]
Hi! I am Lauren and red pandas are my favorite animal.
![image](https://user-images.githubusercontent.com/49538303/186204865-758d759b-9173-4577-823b-e9ecb64ff631.png)

**Comment by justinheadley** [comment]
![6a010535647bf3970b0192aa1c86c1970d-500wi](https://user-images.githubusercontent.com/97985997/186205121-620f4a4c-b0a8-40aa-b7df-ba14d8bb972d.jpg)
Hello, I am Justin and sloths are my favorite animal.

**Comment by anikdas3103** [comment]
Hi I am Anik, and here is Schrodinger's Cat
![download](https://user-images.githubusercontent.com/111896356/186205337-f3dfbeaf-a7e6-4d95-9836-4030382f224a.jpg)

**Comment by Blowi818** [comment]
Hello! I am blowi and i think owls are cool


![OIP](https://user-images.githubusercontent.com/111896348/186205394-502bb95b-dc6e-457c-9932-241d45ea4f37.jpg)

**Comment by JakeSmith1993** [comment]
Hey! I'm Jake my favorite animal are sharks, not exactly cute though.

![image](https://user-images.githubusercontent.com/97986174/186205528-bc0a97a4-269b-493e-b640-0129726b7883.png)

**Comment by edenharper** [comment]
Hey I'm Eden and my favorite animal is my cat Ham
![IMG-5337](https://user-images.githubusercontent.com/111896318/186205711-143b38e2-b61a-451b-9014-426479bebbef.JPG)

**Comment by Steven275** [comment]
Hi, i'm Steven and I am a cat person.
![image](https://user-images.githubusercontent.com/111896459/186205660-203dd368-4dc6-4952-8076-e3f516aaccd7.png)
https://www.thesprucepets.com/understanding-cat-sleep-553961

**Comment by e1000ioa** [comment]
They can fly, swim and walk. They do it all poorly but they do it all anyway. 
I'm Emilio, Ducks are my favorite animal. 
![image](https://user-images.githubusercontent.com/61033492/186205702-87904d08-5f15-48d8-9f0c-0d9b1f279bc3.png)
